python/src/trezorlib/cli/sol.py

```
@@ -53,3 +64,25 @@ def sign_tx(client: "TrezorClient", address: str, raw_tx: str) -> str:
     address_n = tools.parse_path(address)
     transaction = solana.sign_tx(client, address_n, base58.b58decode(raw_tx))
     return transaction.signature.hex()
+
+@cli.command()
+@click.option("-n", "--address", required=True, help=PATH_HELP)
+@click.option("-v", "--message-version", type=ChoiceType(MESSAGE_VERSIONS), default="v0")
+@click.option("-f", "--message-format", type=ChoiceType(MESSAGE_FORMATS), default="ascii")
+@click.option("-d", "--application-domain", default=None, help="32 bytes hex encoded application domain or None")
+@click.argument("message")
+@with_client
+def sign_message(client: "TrezorClient",
```

Ruff (0.8.2)

Missing return type annotation for public function `sign_message`

(ANN201)